pkg/proxy/iptables/proxier_test.go

```diff
@@ -1343,8 +1343,10 @@ func getLine() int {
 }
 
 // assertIPTablesRulesEqual asserts that the generated rules in result match the rules in
-// expected, ignoring irrelevant ordering differences.
-func assertIPTablesRulesEqual(t *testing.T, line int, expected, result string) {
+// expected, ignoring irrelevant ordering differences. By default this also checks the
+// rules for consistency (eg, no jumps to chains that aren't defined), but that can be
+// disabled by passing false for checkConsistency if you are passing a partial set of rules.
```

Comment:
is false used somewhere?

Reply:
It's not used in the commit that introduces that argument, but it gets used in the next commit. (`checkConsistency=false` is only useful with partial syncs; if you're doing a full sync you should always check the consistency.)